src/Entity/Agenda.php

```diff
@@ -86,6 +86,12 @@ class Agenda implements LoggableEntityInterface
      */
     private $agendaBroadcasts;
 
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
```

Comment:
Are 255 characters enough?

Reply:
I would of thought so, however let's change it to type `text`.